icerate.py

```diff
@@ -436,7 +439,7 @@ def print_syntax():
     optdict = dict(optlist)
 
     try:
-        infile = optdict.get("-f", args[0])
+        infile = optdict.get("-f")
```

Comment:
I think we should handle the case that `"-f"` isn't in the `optdict` dictionary. The `IndexError` below is now unreachable.

I'd recommend something like:

```python
try:
    infile = optdict["-f"]
except KeyError:
    sys.stderr.write("A survey filename must be supplied\n")
    sys.exit(1)

```